docs/CHANGELOG.asciidoc

```diff
@@ -101,6 +102,392 @@ Review important information about the {kib} 8.x releases.
 
 include::upgrade-notes.asciidoc[]
 
+[[release-notes-8.19.0]]
+== {kib} 8.19.0
+
+coming::[8.19.0]
+
+Review the following information about the {kib} 8.19.0 release.
+
+[float]
+[[breaking-changes-8.19.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade to 8.19.0, review the breaking changes, then mitigate the impact to your application.
+
+[discrete]
+.Change to variable syntax in {esql} queries
+[%collapsible]
+====
+*Details* +
+Fields and functions variables are now described with `??` in {esql} queries. For value variables, you can continue to use `?` ({kibana-pull}213916[#213916]).
+====
+
+[float]
+[[deprecations-8.19.0]]
+=== Deprecations
+
+There are no deprecations in this version.
+
+[float]
+[[features-8.19.0]]
+=== Features
+{kib} 8.19.0 adds the following new and notable features.
+
+Alerting::
+* Adds the ability to schedule reports on a recurring basis and view previously scheduled reports ({kibana-pull}224849[#224849], {kibana-pull}221028[#221028]).
+* Adds four dedicated case analytics indices that allow users to build dashboards and metrics using case data. These indices are automatically created when {kib} starts up and are updated periodically with cases, comments, attachments, and activity data ({kibana-pull}223405[#223405]).
```

Comment:
Need to pull this. Was removed from 8.19/9.1.

cc: @michaelolo24 @paulewing @tiamliu @adcoelho 

```suggestion
```